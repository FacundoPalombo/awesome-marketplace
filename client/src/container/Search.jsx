import React from "react";
import { Helmet } from "react-helmet";
import styles from "./Search.module";

export function Search() {
  return (
    <>
      <Helmet>
        <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
        <title>Busqueda de productos</title>
      </Helmet>
      <section id="search" className={styles.main}></section>
    </>
  );
}
