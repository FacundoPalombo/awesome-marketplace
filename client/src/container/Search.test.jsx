import React from "react";
import { renderWithRouter } from "utils/tests/renderWithRouter";
import { Search } from "./Search";

describe("GIVEN Search Component", () => {
  describe("WHEN page is rendered", () => {
    it("THEN should match to snapshot", () => {
      const { asFragment } = renderWithRouter(<Search />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
