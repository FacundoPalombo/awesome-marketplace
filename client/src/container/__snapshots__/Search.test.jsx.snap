// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN Search Component WHEN page is rendered THEN should match to snapshot 1`] = `
<DocumentFragment>
  <section
    class="main"
    id="search"
  />
</DocumentFragment>
`;
